eng/templates/stages/deploy.yaml

```diff
@@ -197,11 +220,7 @@ stages:
       displayName: Test Azure CLI auth
 
     - powershell:
-        .\darc\darc.exe get-default-channels --source-repo arcade-services --ci -t "$(GetAuthInfo.FederatedToken)" --bar-uri "$(GetAuthInfo.BarUri)" --debug
-      displayName: Test Federated token auth
-
-    - powershell:
-        .\darc\darc.exe get-default-channels --source-repo arcade-services --ci --password "$(scenario-test-maestro-token)" --bar-uri "$(GetAuthInfo.BarUri)" --debug
+        .\darc\darc.exe get-default-channels --source-repo arcade-services --ci --password "$(GetAuthInfo.Token)" --bar-uri "$(GetAuthInfo.BarUri)" --debug
```

Comment:
Do we need to remove the option for federated tokens then?

Reply:
We could, since I'm guessing we were the only ones using it